llvm/lib/Target/AMDGPU/GCNSubtarget.h

```diff
@@ -967,6 +967,7 @@ class GCNSubtarget final : public AMDGPUGenSubtargetInfo,
 
   bool hasLDSFPAtomicAddF32() const { return GFX8Insts; }
   bool hasLDSFPAtomicAddF64() const { return GFX90AInsts; }
+  bool hasXnackReplay() const { return GFX8Insts; }
```

Comment:
We already have a field SupportsXNACK for this which is hooked up to the "xnack-support" target feature.